### ISSUE TITLE
[QUESTION] Is it possible to use custom Flutter SDK?

### ISSUE THREAD

**Issue by thaidmfinnick** [subosito/flutter-action#308]
Hi, I'm building Flutter App with Github Actions (of course cannot live without `flutter-action`), my problem is I have been customized some files in Flutter Engine, Flutter SDK to develop my app so I cannot use pure Flutter SDK as `flutter-action` provided when building.

Can I do some workaround to use custom flutter sdk in `flutter-action`?

Thank you!

**Comment by bartekpacia** [comment]
Hey – it's not possible currently, but this sounds like a valid feature request to me. An example is `actions/setup-java` Action and how it allows for installation of different JDK distributions.

Please feel free to write down how'd you like this feature to look like, and make a PR.

**Comment by thaidmfinnick** [comment]
Thank you for your quick response.

I want to add my fork repo of Flutter SDK into GHA, and if possible with hash commit, and demo in my idea like that:
```
steps:
  - name: Clone repository
    uses: actions/checkout@v4
  - name: Set up Flutter
    uses: subosito/flutter-action@v2
    with:
      git: https://github.com/thaidmfinnick/flutter 
      ref: 34f9afg
  - run: flutter --version
  ```
  
  Actually I haven't understand how Flutter SDK, Engine was built and join together on GHA so I'm not sure it's possible.
  Look forward to seeing your response, tks!

**Comment by bartekpacia** [comment]
related #331

**Comment by bartekpacia** [comment]
Closed by #334
